### DISCUSSION TITLE
Limitations with the assignment of bookmarks

### DISCUSSION THREAD

**Discussion by alejandrolga** [directus/directus#23582] [Feature Requests]
### Summary

Limitations with the assignment of bookmarks, as this feature is restricted to a single user, a role, or all users.

### Motivation

The functionality is very simple: to enable the assignment of bookmarks to multiple users and policies.

### Detailed Design

Our Directus project is currently facing limitations with the assignment of bookmarks, as this feature is restricted to a single user, a role, or all users. We have very specific roles, such as Leaders and Operators, but within these categories, there are multiple subtypes with unique policies. For example, there are 10 different types of operators, each governed by its own specific policy, alongside a general policy that applies to all. Similarly, leaders may have different policy-based assignments, such as "Leader - Design" or "Leader - 3D," each of which might include multiple users. The current bookmark assignment system is inadequate because it doesn’t account for these nuanced differences. Enabling the assignment of bookmarks to multiple users based on specific roles and policies would greatly enhance the functionality and flexibility of the system.

### Must Have:
- A. An option to assign bookmark with a specific policy.
- B. An option to assign bookmark with multiple users.

### Should Have:
- C. An option to assign bookmark with a multiple policies.

### Drawbacks

This enhancement improves the user experience and makes the application more flexible and adaptable in terms of assignments.

### Alternatives

There is a limitation for companies that utilize the same structure in their permissions, as the application restricts the assignment of permissions to specific roles when attempting to use bookmarks.

### Adoption Strategy

This change is straightforward and will not significantly impact the software. It serves as an enhancement, offering additional functionality. Users who choose not to utilize this feature will not be affected, while those who do will benefit from increased flexibility in assigning bookmarks with more options

**Comment by renansantos** [comment]
I really need this.
When policies was added, that was the first thing i thought!
Bookmarks by policies will increase significantly the flexibility of the feature.
I even tried to edit the raw content of the scope to use a policy, but it didn't work.
